util/__init__.py

```diff
@@ -130,8 +130,8 @@ def setup(args, extra):
         repo_main_args = (
             ['init', '-u', ADX_MANIFEST_URL, '-m', manifest],
             ['sync', '--force-sync'],
-            ['forall', '-c', 'git', 'checkout', 'master'],
-            ['forall', 'ckan', '-c', 'git', 'checkout', 'refs/tags/ckan-2.8.2'],
+            ['forall', '-c', 'git', 'checkout', 'development'],
+            ['forall', 'ckan', '-c', 'git', 'checkout', 'refs/tags/ckan-2.9.1'],
             ['forall', 'ckanext-scheming', '-c', 'git', 'checkout', 'validator'],
```

Comment:
```
            ['forall', '-c', 'git', 'checkout', 'master'],
            ['forall', '-c', 'git', 'checkout', 'development'],
            ['forall', 'ckan', '-c', 'git', 'checkout', 'refs/tags/ckan-2.8.2'],
            ['forall', 'ckanext-scheming', '-c', 'git', 'checkout', 'validator'],
```
I think all those 3 lines should go away. We have defined git refs for setup in the manifest. I'll push a commit cleaning this up.

Reply:
Yes my recollection is that the refs have to stay since repo doesn't checkout local branches for you.